packages/suite-build/configs/desktop.webpack.config.ts

```diff
@@ -17,9 +17,8 @@ const baseDirUI = getPathForProject('desktop-ui');
 const baseDir = getPathForProject('desktop');
 
 const config: webpack.Configuration = {
-    // Electron 38 runs on Chromium 140 https://github.com/electron/electron/releases/tag/v38.0.0
-    // but we are limited to 138 (supported by latest browserslist, as included by latest webpack)
-    target: 'browserslist:Chrome >= 138',
+    // Electron 35 runs on Chromium 134 https://www.electronjs.org/blog/electron-35-0#stack-changes
+    // but we are limited to 133 (supported by latest browserslist, as included by latest webpack)
```

Comment:
the target should not be removed. On the other hand, the comment is unnecessary because now our webpack supports 134
```suggestion
    target: 'browserslist:Chrome >= 134',
```

Reply:
9546d4d06a6841d59487e7b993559cd00f959fed